src/pages/Possibility/KivaContentBlock.vue

```diff
@@ -28,7 +28,7 @@
 					</div>
 				</div>
 				<div class="columns large-5 show-for-large">
-					<kv-responsive-image :images="images" />
+					<kv-responsive-image :images="images" alt="Images of possibility" />
```

Comment:
This can be alt="" since "images of possibility" doesn't provide much useful context to a user. We'd have to pass it in with the images array if there was something we really wanted to call out.